aredis_om/model/model.py

```diff
@@ -765,6 +765,10 @@ def sort_by(self, *fields: str):
             return self
         return self.copy(sort_fields=list(fields))
 
+    async def page(self, offset, count):
+        query = self.copy(page_size=count, offset=offset)
+        return await query.execute(exhaust_results=False)
```

Comment:
I tried this locally and the `execute` function doesn't seem to use the `page_size` and `offset` values in the way that we maybe originally thought.  It would seem to use them to manage an internal cache? See: https://github.com/redis/redis-om-python/blob/dcd84e0ab2686a3c6ff7f6c6a43ec49f4b65655f/aredis_om/model/model.py#L723